src/utils/saml.ts

```diff
@@ -610,11 +613,11 @@ export const getPreValidateResponse = (
     );
 
   /* LOGGING INFOS:
-    having the idpIssuer and requestId as data here we leverage multiple advantages:
-    1. we can query based on the idp and display graphs about errors/usage
-    2. we know what idp is causing the error
-    3. having the requestId it's possible to analyze further the problem encountered
-  */
+            having the idpIssuer and requestId as data here we leverage multiple advantages:
+            1. we can query based on the idp and display graphs about errors/usage
+            2. we know what idp is causing the error
+            3. having the requestId it's possible to analyze further the problem encountered
+          */
```

Comment:
Wrong indentation. Use `/**` instead of `/*` comment.
```suggestion
   * having the idpIssuer and requestId as data here we leverage multiple advantages:
   * 1. we can query based on the idp and display graphs about errors/usage
   * 2. we know what idp is causing the error
   * 3. having the requestId it's possible to analyze further the problem encountered
   */
```